recipe/unzip.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"archive/zip"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+// Unzip extracts a ZIP file located at src into a specified directory. It uses package 'archive/zip' internally.
+// Giving an empty string as the target dir will unzip the contents in the current directory.
+func Unzip(src, targetDir string) error {
```

Comment:
I thought maybe we should move the Unzip function to the Downloader and turn it into a Method. That would make the Unzip function still testable but would avoid having a unnecessarily public function inside the main scope. So in the main function we could do something like:

```
...
zip := downloader.Download()
unzipped := downloader.Unzip(zip)
...
```

What do you think?